pyro/contrib/autoguide/__init__.py

```diff
@@ -561,6 +562,49 @@ def _loc_scale(self, *args, **kwargs):
         return loc, scale
 
 
+class AutoTransformedNormal(AutoContinuous):
+    """
+    This implementation of :class:`AutoContinuous` uses a Transformed
+    Diagonal Normal distribution via an IAF to construct a guide
```

Comment:
uses a Diagonal Normal distribution transformed by an Inverse Autoregressive Flow

Reply:
also it should probably just becalled something like AutoIAFNormal